helm/grafana/templates/grafana-database/bucket.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.grafanaDatabase.enabled }}
+# Database backup storage
+apiVersion: objectstorage.giantswarm.io/v1alpha1
+kind: Bucket
+metadata:
+  annotations:
+    meta.helm.sh/release-name: {{ .Chart.Name }}
```

Comment:
I do not think we need the annotations?

Reply:
I'm not sure but as they're also present on other buckets, I'd rather have them here as well.

Reply:
SUre but we need it for backups only right?